build/go/benchmark_test.sh

```diff
@@ -6,4 +6,4 @@ ulimit -a
 
 # Looks like CGO has to be enabled for -race if we're using go modules
 # Let benchmark failures fail the build since the plot conversion just renders empty results
-CGO_ENABLED=1 go test ./... -race -run=Bench -bench=. -benchtime=5s -v --tags=race_tests -cpu=12 | tee benchmark.out && test ${PIPESTATUS[0]} -eq 0
+CGO_ENABLED=1 go test ./... -race -run=Bench -bench=. -v --tags=race_tests | tee benchmark.out && test ${PIPESTATUS[0]} -eq 0
```

Comment:
Honestly, we couldn't find any good reason for them to continue to exist, and they were causing tests to run for much longer than seemed necessary. Benchtime defaults to 1s, so the tests run for only 1/5 of the time they were. And specifying 12 CPUs doesn't seem that useful, since the actions runners will certainly not have 12 CPUs available to use, and this one will default to `GOMAXPROCS` anyhow, to max out its resources.